### PR TITLE
Add more docs about what is going on in SSHManager

### PR DESCRIPTION
**Pull request by jakebolewski**
While trying to work on a bug, I decided that future-self will not have to look at the ssh manpage anymore.

### CODE REVIEW

**Comment by amitmurthy** [comment]
:+1:

File: base/managers.jl [hunk 1]
```diff
@@ -153,18 +186,33 @@ let tunnel_port = 9201
...
+    localport = next_tunnel_port()
+    # if we cannot do port forwarding, bail immediately
+    # the connection is forwarded to `port` on the remote server over the local port `localp`
+    # the -f option backgrounds the ssh session
+    # `sleep 60` command specifies that an alloted time of 60 seconds is allowed to start the
+    # remote julia process and estabilish the network connections specified the the process topology.
+    # If no connections are made within 60 seconds, ssh will exit and an error will be printed on the
+    # process that launched the remote process.
+    ssh = `ssh -T -a -x -o ExitOnForwardFailure=yes`
+    while !success(detach(`$ssh -f $sshflags $user@$host -L $localp:$bind_addr:$port sleep 60`)) && cnt > 0
```
Comment (amitmurthy) [inline base/managers.jl]: This should be `$localport` not `localp`

Probable cause of https://github.com/JuliaLang/julia/issues/13696

Comment (jakebolewski) [inline base/managers.jl]: Whoops, thanks.  #13696 is not caused by this as that report was from yesterday.

Comment (amitmurthy) [inline base/managers.jl]: fixed in https://github.com/JuliaLang/julia/pull/13706

**Pull request merged**